openapi_tester/schema_converter.py

```diff
@@ -8,7 +8,7 @@ class SchemaToPythonConverter:
     """ This class is used both by the DocumentationError format method and the various test suites """
 
     def __init__(self, schema: dict, with_faker: bool = False):
-        while any([keyword in schema for keyword in ["allOf", "oneOf", "anyOf"]]):
```

Comment:
this is a regression: https://www.w3schools.com/python/ref_func_any.asp

Reply:
How so? It should work exactly the same - the list comprehension is just redundant as any already accepts generators